prow/config/jobs/istio.yaml

```diff
@@ -221,6 +221,194 @@ jobs:
     requirements: [kind]
     timeout: 4h
 
+
+
+  - name: unit-tests-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    command: [make, -e, "T=-v", build, localTestEnv, test, binaries-test]
+
+  - name: racetest-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    command: [make,  -e, "T=-v", localTestEnv, racetest]
+
+  - name: codecov-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    command: [make, coverage-diff]
+
+  - name: release-test-new
+    modifiers: [optional, hidden]
+    image: gcr.io/istio-testing/build-tools:2019-10-02T14-57-08
+    type: presubmit
+    command: [prow/release-test.sh]
```

Comment:
This one won't work, missing some stuff like `fpm`

Reply:
I'm expecting nearly every new job to fail. But having all the jobs there, we can tackle issues one by one and gradually gets chunks of jobs to work.